### PR TITLE
Fixing allPossibleMigStrategiesAreNone helm chart helper.

### PR DESCRIPTION
**Pull request by jakubkrzykowski**
`configMapName` is the empty string `""` by default and it breaks `allPossibleMigStrategiesAreNone` function with default values.

See https://github.com/NVIDIA/k8s-device-plugin/issues/670 for details.

### CODE REVIEW

**Review by elezar — COMMENTED** [review]
Sorry, these were pendings.

File: deployments/helm/nvidia-device-plugin/templates/_helpers.tpl [hunk 1]
```diff
@@ -130,7 +130,7 @@ Check if migStrategy (from all possible configurations) is "none"
   {{- if ne .Values.migStrategy "none" -}}
     {{- $result = false -}}
   {{- end -}}
-{{- else if ne (include "nvidia-device-plugin.configMapName" .) "true" -}}
+{{- else if ne (include "nvidia-device-plugin.configMapName" .) "" -}}
```
Comment (elezar) [inline deployments/helm/nvidia-device-plugin/templates/_helpers.tpl:133]: Let's use the following instead:
```suggestion
{{- else if eq (include "nvidia-device-plugin.hasConfigMap" .) "true" -}}
```

**Review by elezar — CHANGES_REQUESTED** [review]

**Review by elezar — APPROVED** [review]
Thanks @jakubkrzykowski. These look good now.

I rebased and incorporated the changes from #749

**Comment by adammw** [comment]
having problems with this using the built in config.map option - since a configMapName is set it immediately goes to `$result = false` instead of falling through to evaluate the config.map profiles

**Pull request merged**